apollo-gradle-plugin/src/test/groovy/com/apollographql/apollo/gradle/unit/ApolloAndroidPluginSpec.groovy

```diff
@@ -73,8 +73,7 @@ class ApolloAndroidPluginSpec extends Specification {
 
     then:
     project.android.sourceSets.all { sourceSet ->
-      assert (sourceSet.extensions.findByName("graphql")) != null
```

Comment:
Really small comment. I find having hard coded `graphql` in tests is better in terms of quality assurance.

Imagine a case where somebody changes `GraphQLSourceDirectorySet.NAME` parameter, that will be a breaking change. The previous version of these tests will catch it whereas the new one will just accept them.